package.json

```diff
@@ -44,33 +55,30 @@
   },
   "scripts": {
     "start": "node ./scripts/start-dev-server.js",
-    "setup": "bash ./scripts/setup.sh",
+    "setup": "npm ci && npx husky install",
```

Comment:
`prepare` could do the job. See docs -- https://docs.npmjs.com/cli/v10/using-npm/scripts#npm-ci